README.md

```diff
@@ -13,10 +13,57 @@ npm install cf-buttons
 npm install cf-expandables
 ```
 
+Want some boilerplate?
+
+```sh
+npm install -g yo generator-cf
+mkdir my-new-project && cd $_
+yo cf
+```
+
 | buttons | core | expandables | forms | grid | icons | layout | pagination | typography |
 |:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
 | [![npm](https://img.shields.io/npm/v/cf-buttons.svg?style=flat-square)](https://www.npmjs.com/package/cf-buttons)  | [![npm](https://img.shields.io/npm/v/cf-core.svg?style=flat-square)](https://www.npmjs.com/package/cf-core)  | [![npm](https://img.shields.io/npm/v/cf-expandables.svg?style=flat-square)](https://www.npmjs.com/package/cf-expandables)  | [![npm](https://img.shields.io/npm/v/cf-forms.svg?style=flat-square)](https://www.npmjs.com/package/cf-forms)  | [![npm](https://img.shields.io/npm/v/cf-grid.svg?style=flat-square)](https://www.npmjs.com/package/cf-grid)  | [![npm](https://img.shields.io/npm/v/cf-icons.svg?style=flat-square)](https://www.npmjs.com/package/cf-icons)  | [![npm](https://img.shields.io/npm/v/cf-layout.svg?style=flat-square)](https://www.npmjs.com/package/cf-layout)  | [![npm](https://img.shields.io/npm/v/cf-pagination.svg?style=flat-square)](https://www.npmjs.com/package/cf-pagination)  | [![npm](https://img.shields.io/npm/v/cf-typography.svg?style=flat-square)](https://www.npmjs.com/package/cf-typography)  |
 
+## Contributing
+
+### Building components locally
+
+```sh
+git clone git@github.com:cfpb/capital-framework.git
+cd capital-framework
+npm install
+npm run build
```

Comment:
I love this. I opened an issue in the standards to see how folks feel about using it on other projects here: https://github.com/cfpb/front-end/issues/101

:can-of-worms: